core/root/config/templates/cluster.yaml

```diff
@@ -19,6 +19,21 @@ amiId: "{{.AmiId}}"
 # Container Linux has automatic updates https://coreos.com/os/docs/latest/update-strategies.html. This can be a risk in certain situations and this is why is disabled by default and you can enable it by setting this param to false.
 disableContainerLinuxAutomaticUpdates: true
 
+# Override the CloudFormation logical sub-stack names of control plane, etcd and/or network.
+# Changing these names causes CF to delete the old and create a new stack and thus deleting and recreating all components in the stack.
+#
+# From kube-aws 0.11.x and onwards, separate networking and etcd CF stacks have been introduced.
+# Because CF does not support migration of resources between stacks (or renaming stacks), a new VPC will be created if you upgrade an existing 0.10.x cluster.
+# Creating a new VPC and deleting the old one can cause a lot of issues and might not be desired.
+# To make this migration more feasible it is posible to change the name of the network stack to the old controleplane stack to keep the vpc components in the same stack. This way CF will not create a new vpc.
+# In this scenario the name of the networkstack becomes "ControlPlane". (which is the name of the stack which holds the vpc components in a kube-aws pre 0.11 cluster)
+# For clearity it is advised also rename the stack of the controlPlane components to, for example, "masters"
+# Note that this will keep some legacy naming around in your configs and CF stacks...
+#stackNameOverride:
```

Comment:
Nit but:

`
cloudformation:
  stackNameOverrides:
    ...
```

sounds slightly nicer to me because it expresses you can one or more stack names customized, and it works in term of CloudFormation.